dynawo/sources/Models/CPP/ModelNetwork/DYNModelTwoWindingsTransformer.cpp

```diff
@@ -1368,21 +1371,23 @@ ModelTwoWindingsTransformer::P2(const double& ur1, const double& ui1, const doub
 void
 ModelTwoWindingsTransformer::evalG(const double& t) {
   int offset = 0;
+  double ur1Val = 0.;
+  double ui1Val = 0.;
+  double ur2Val = 0.;
+  double ui2Val = 0.;
+  if (currentLimits1_ || currentLimits2_ || modelPhaseChanger_) {
```

Comment:
Do a similar optimization in evalNodeInjection

Reply:
Solved